packages/sdk-codegen/src/specConverter.spec.ts

```diff
@@ -402,8 +402,8 @@ const swagger = JSON.parse(swaggerFrag)
 const api = JSON.parse(openApiFrag)
 
 const config = TestConfig()
-const swaggerFile = `${config.rootPath}/spec/Looker.4.0.json`
-const apiFile = `${config.rootPath}/spec/Looker.4.0.oas.json`
+const swaggerFile = `${config.rootPath}spec/Looker.4.0.json`
+const apiFile = `${config.rootPath}spec/Looker.4.0.oas.json`
```

Comment:
testUtils now has a `rootFile()` function we can use to fix this up in a future PR